third_party/mbedtls/CMakeLists.txt

```diff
@@ -75,6 +92,7 @@ target_sources(mbedcrypto PRIVATE
     repo/library/ssl_ticket.c
     repo/library/ssl_tls.c
     repo/library/threading.c
+    repo/library/timing.c
```

Comment:
What is the use case for including `timing.c`?

Reply:
otbr-commissioner uses the timer API in mbedtls. Adding this without defining `TIMING_C` in mbedtls-config.h will not affect OpenThread.